modules/api/src/main/java/org/apache/ignite/schema/ColumnType.java

```diff
@@ -163,6 +178,38 @@ public static DecimalColumnType decimalOf() {
         );
     }
 
+    /**
+     * Returns timezone-free type representing a time of day in hours, minutes, seconds,
+     * and fractional seconds depending on type precision.
+     *
+     * @param precision Fractional seconds meaningful digits. Allowed values are 0-9 for second to nanosecond precision.
+     * @return Native type.
+     */
+    public static TemporalColumnType time(int precision) {
```

Comment:
ColumnType is a part of public API, thus I would prefer to use enum for precisions. Otherwise we 1) need to improve javadoc, since currently it's unclear what actually precision=4 means, and 2) throw an IllegalArgumentException in case someone pass a precision that out of valid range.